frontend/src/features/counter/Counter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  decrement,
+  increment,
+  incrementByAmount,
+  incrementAsync,
+  selectCount,
+} from './counterSlice';
+import styles from './Counter.module.css';
+
+export function Counter() {
```

Comment:
:+1: for using function as opposed to class components

Reply:
tbh this is auto generated, I didnt add thing, i left it in for people who start off to maybe have something to work off of